robottelo/ui/activationkey.py

```diff
@@ -175,6 +175,27 @@ def add_host_collection(self, name, host_collection_name):
             raise UINoSuchElementError(
                 "Couldn't find activation key '{0}'".format(name))
 
+    def remove_host_collection(self, name, host_collection_name):
+        """Remove an existing Host Collection from Activation Key."""
+        # find activation key
+        activation_key = self.search(name)
```

Comment:
Don't mind and thought about that in the past, but it can be too costly with almost zero outcome from business value perspective, but sure, github issue will never hurt =)